### PR TITLE
Add a batch validate API to the data service

### PR DESCRIPTION
**Pull request by axmb**

### CODE REVIEW

File: data-serving/data-service/api/openapi.yaml [hunk 1]
```diff
@@ -117,6 +117,22 @@ paths:
           $ref: "#/components/responses/422"
         "500":
           $ref: "#/components/responses/500"
+  /cases/batchValidate:
```
Comment (khmoran) [inline data-serving/data-service/api/openapi.yaml:120]: will we be able to remove the validate_only param on the other endpoint once we're using this?

Comment (axmb) [reply data-serving/data-service/api/openapi.yaml:114]: Yep, should -- the only current user will be migrating to the batch API.

**Comment by axmb** [comment]
Retroactively taggin #584

**Comment by axmb** [comment]
[Bug fixes](https://trello.com/c/DKJztzhL/276-bug-fixes)

**Pull request merged**